bin/flutter

```diff
@@ -41,6 +41,8 @@ if [ ! -f "$SNAPSHOT_PATH" ] || [ ! -f "$STAMP_PATH" ] || [ `cat "$STAMP_PATH"`
   echo $REVISION > "$STAMP_PATH"
 fi
 
+FLUTTER_DEV=1
```

Comment:
Please remove.
(Even if you want to use this locally, you shouldn't be committing this.)